packages/website/package.json

```diff
@@ -20,12 +20,16 @@
     "@docusaurus/remark-plugin-npm2yarn": "2.0.0-beta.21",
     "@docusaurus/theme-common": "2.0.0-beta.21",
     "@mdx-js/react": "1.6.22",
+    "@typescript-eslint/parser": "^5.31.0",
     "@typescript-eslint/website-eslint": "5.31.0",
     "clsx": "^1.1.1",
     "eslint": "*",
+    "json-schema": "^0.4.0",
+    "json-schema-to-typescript": "^11.0.1",
     "json5": "^2.2.1",
     "konamimojisplosion": "^0.5.1",
     "lzstring.ts": "^2.0.2",
+    "prettier": "*",
```

Comment:
I believe this would pull in the latest version of Prettier every time, even when we locked the version in the main package?

Reply:
Hmm, I think that's happening regardless from the `json-schema-to-typescript` dependency. Otherwise I would have thought it'd not impact the version?

Reply:
Transitive dependencies are different from workspace dependencies, afaict. But I haven't checked which version we are pulling in right now.

Reply:
We'll have these all bumped to the latest soon anyway. 🚀